lib/marquee.dart

```diff
@@ -99,6 +99,7 @@ class Marquee extends StatefulWidget {
     this.blankSpace = 0.0,
     this.velocity = 50.0,
     this.pauseAfterRound = Duration.zero,
+    this.stopAfterMaxRound = 0,
```

Comment:
Right now, we're using a normal `int` value (`0`) to indicate a special condition (make infinite rounds). I think we should use `null` for that instead of `0` – i.e. using a special value to indicate a special condition.
Also, we should add an `assert`, something like `assert(stopAfterMaxRound == null || stopAfterMaxRound > 0)`.

Reply:
done